traffic_ops/traffic_ops_golang/api/api.go

```diff
@@ -369,6 +382,78 @@ func (inf *APIInfo) Close() {
 	}
 }
 
+// CreateInfluxClient onstructs and returns an InfluxDB HTTP client, if enabled and when possible.
+// The error this returns should not be exposed to the user; it's for logging purposes only.
+//
+// If Influx connections are not enabled, this will return `nil` - but also no error. It is expected
+// that the caller will handle this situation appropriately.
+func (inf *APIInfo) CreateInfluxClient() (c *influx.Client, e error) {
```

Comment:
I don't know that we have an official stance as a project, but I think the majority of the Go community discourages named returns. It makes it non-obvious what's being returned where. You could set `e` anywhere, and `return` anywhere else, and then it's hard for a reader to follow that and figure out what `e` and `c` are being returned at any given `return`.

I won't -1, if you feel strongly about it, but I do think it hinders readability, and it's better to avoid, so future readers can easily see exactly what's being returned at every point.

Reply:
I don't really care at all, just seems like the thing to do when your first action would otherwise be to declare the things you're returning.

Reply:
just something i read: "Naked return statements should be used only in short functions. They can harm readability in longer functions."

this function doesn't look that short...